### DISCUSSION TITLE
Honor user's tab size preference in markdown code blocks

### DISCUSSION THREAD

**Discussion by rschristian** [community/community#20578] [Projects and Issues] [answered]
Ideally a user's tab size preference should also apply to code blocks within markdown. Currently they do not, and are set to an automatic (& static) width of 8 spaces.

Here's an example:

```js
while (true) {
	x();
}
```

I personally have my tab size set to 4 spaces, and yet, that shows as 8.

**Comment by y9su** [comment]
Hello, please read this- https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket
You can request that to github.
Please mark this as an answer if it helped!

**Reply by rschristian** [reply]
Er, nope. Sounds like you might need to read that link first:

> If your account uses a paid GitHub product or you are a member of an organization that uses a paid product, you can directly contact GitHub Support. If your account uses GitHub Free, you can speak to GitHub users and staff on the [GitHub Support Community](https://github.community/) for most issues, and you can contact GitHub Support to report account, security, and abuse issues.

This isn't an account, security, or abuse issue, and my account does not use a paid GitHub product, ergo, GitHub Support is not relevant.

**Comment by aradalvand** [comment]
Agreed. I find it unfathomable that this setting is not respected in code snippets. What exactly was the point of this option in the first place if it was not going to apply to code snippets within comments?

**Comment by BenjaminAster** [comment, marked as answer]
I absolutely agree, but nonetheless, this is a duplicate of [#8067](https://github.com/github-community/community/discussions/8067).

**Reply by rschristian** [reply]
Apologies, didn't come up in my search.
